From f3e24df61b30fc1acff2fc5f41633b829b7e3dc4 Mon Sep 17 00:00:00 2001
From: minhmc2007 <quangminh21072010@gmail.com>
Date: Sun, 4 Oct 2026 09:45:05 +0700
Subject: [PATCH] gapps: drop the prebuilt_apex apps property

error: vendor/gapps/x86_64/Android.bp:90:9: unrecognized property "apps"

apps on prebuilt_apex is newer than the Soong in Android 15, and gapps
added x86_64 support in 74fba8f "Reland x86_64 support", 2026-07-07, against
a Soong that has it. That commit is the only one touching x86_64 on
cinnamonbun, so there is no older revision of this branch to pin instead.

The property only associates the apex with the PrebuiltGmsCoreVic module from
dummy-apps. Dropping it leaves the apex building and installing, which is all
a phone build needs.
---
 arm/Android.bp    | 1 -
 arm64/Android.bp  | 1 -
 x86_64/Android.bp | 1 -
 3 files changed, 3 deletions(-)

diff --git a/arm/Android.bp b/arm/Android.bp
index 2fe725d..e4c7cc4 100644
--- a/arm/Android.bp
+++ b/arm/Android.bp
@@ -94,6 +94,5 @@ prebuilt_apex {
     owner: "gapps",
     src: ":com.google.android.gmssystem.prodvic.apex-genrule",
     filename: "com.google.android.gmssystem.prodvic.apex",
-    apps: ["PrebuiltGmsCoreVic"],
     product_specific: true,
 }
diff --git a/arm64/Android.bp b/arm64/Android.bp
index 2e79c2b..ea0be12 100644
--- a/arm64/Android.bp
+++ b/arm64/Android.bp
@@ -134,6 +134,5 @@ prebuilt_apex {
     owner: "gapps",
     src: ":com.google.android.gmssystem.prodvic.apex-genrule",
     filename: "com.google.android.gmssystem.prodvic.apex",
-    apps: ["PrebuiltGmsCoreVic"],
     product_specific: true,
 }
diff --git a/x86_64/Android.bp b/x86_64/Android.bp
index 5171aa9..b721bb8 100644
--- a/x86_64/Android.bp
+++ b/x86_64/Android.bp
@@ -87,6 +87,5 @@ prebuilt_apex {
     owner: "gapps",
     src: ":com.google.android.gmssystem.prodvic.apex-genrule",
     filename: "com.google.android.gmssystem.prodvic.apex",
-    apps: ["PrebuiltGmsCoreVic"],
     product_specific: true,
 }
